### PR TITLE
Rebased #1551

### PR DESCRIPTION
**Pull request by jcspader**
Rebased #1551

### CODE REVIEW

**Comment by Arkni** [comment]
Why closing the PR ?

**Comment by jcspader** [comment]
@Arkni  @staabm It's done now.

File: src/localization/messages_pt_BR.js [hunk 1]
```diff
@@ -3,8 +3,9 @@
  * Locale: PT (Portuguese; português)
  * Region: BR (Brazil)
  */
-$.extend( $.validator.messages, {
-	required: "Este campo &eacute; obrigat&oacute;rio.",
+$.extend($.validator.messages, {
```
Comment (Arkni) [inline src/localization/messages_pt_BR.js]: Please add one space after `(` and one before `)` in all lines in this file.
See http://contribute.jquery.org/style-guide/js/#spacing

**Comment by jcspader** [comment]
@Arkni I added one space after ( and one before )

File: src/localization/messages_pt_BR.js [hunk 2]
```diff
@@ -14,16 +15,60 @@ $.extend( $.validator.messages, {
...
 	rangelength: $.validator.format( "Por favor, forne&ccedil;a um valor entre {0} e {1} caracteres de comprimento." ),
 	range: $.validator.format( "Por favor, forne&ccedil;a um valor entre {0} e {1}." ),
 	max: $.validator.format( "Por favor, forne&ccedil;a um valor menor ou igual a {0}." ),
 	min: $.validator.format( "Por favor, forne&ccedil;a um valor maior ou igual a {0}." ),
-	nifES: "Por favor, forne&ccedil;a um NIF v&aacute;lido.",
+
+	// Metodos Adicionais
+	maxWords: $.validator.format(" Por favor, forne&ccedil;a com {0} palavras ou menos." ),
+	minWords: $.validator.format(" Por favor, forne&ccedil;a pelo menos {0} palavras." ),
+	rangeWords: $.validator.format(" Por favor, forne&ccedil;a entre {0} e {1} palavras." ),
```
Comment (Arkni) [inline src/localization/messages_pt_BR.js]: >this change is for the 3 lines above

This should be one space after the `(` not after double quote.

**Comment by jcspader** [comment]
@Arkni  The lines are changed

**Comment by Arkni** [comment]
Looks good to me :)

The only thing you need to do now is squashing your commits into one, and prefix the new commit with `Localisation: `.

I have to go to sleep :sleeping: 
Will recheck tomorrow :)

**Comment by staabm** [comment]
See http://symfony.com/doc/current/contributing/code/patches.html#rework-your-patch for a tutorial on how to squash commits.

You only need the very last chapter of the pages.

**Comment by jcspader** [comment]
@staabm I tried, but not change anything.

**Comment by Arkni** [comment]
Here are some tutos that can help you:
* https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history
* https://schacon.github.io/gitbook/4_interactive_rebasing.html

**Comment by jcspader** [comment]
@Arkni, @staabm 
Guys, I changed. Please check it.

**Comment by staabm** [comment]
@jcspader doesnt look that bad ;-).

only thing missing is:

> The only thing you need to do now is squashing your commits into one, and prefix the new commit with Localisation:.

Thx

**Comment by jcspader** [comment]
@staabm I forgot this. I changed the message and included the prefix.

**Comment by Arkni** [comment]
LGTM :)

Brahim ARKNI
+212 648 56 31 64

<https://ma.linkedin.com/pub/brahim-arkni/89/64/a50>
<https://twitter.com/BrahimArkni>

**Comment by jcspader** [comment]
\0/ I got it!

Guys, Please, update the instructions for developers, because when i did not had these requirements.

**Comment by staabm** [comment]
@jcspader feel free to provde a PR to the contribution guidelines file, in case something is not clear.

https://github.com/jzaefferer/jquery-validation/blob/master/CONTRIBUTING.md

**Comment by staabm** [comment]
@jcspader congratulations for unlocking the next level of your GIT fu ;-)

**Pull request merged**